python-sdk/nuscenes/map_expansion/map_api.py

```diff
@@ -683,18 +704,25 @@ def __init__(self,
 
     def render_centerlines(self,
                            resolution_meters: float,
-                           figsize: Union[None, float, Tuple[float, float]] = None) -> None:
+                           figsize: Union[None, float, Tuple[float, float]] = None,
+                           bitmap: Optional[BitMap] = None) -> None:
```

Comment:
You might want to return the figure handle so that folks can work with it downstream if desired.

Reply:
Good idea. I added it to all render methods.